### ISSUE TITLE
Accidental Typing Information

### ISSUE THREAD

**Issue by theneverstill** [ptmt/react-native-macos#118]
In /Libraries/Utilities/throwOnWrongReactAPI.js

function throwOnWrongReactAPI

includes "key" parameter typing information ": string".

This should be removed.

**Comment by theneverstill** [comment]
Typing information actually appears to be in a number of files, including:

/Libraries/JavaScriptAppEngine/Initialization/InitializeJavaScriptAppEngine.js

**Comment by ptmt** [comment]
Yes, that's because React Native by Facebook originally uses Packager and Babel presets called `react-native`, which strips the types before executing. What do you do exactly that you get problems with these type annotations?

**Comment by theneverstill** [comment]
At the end of the day, I'm trying to setup a react-native jest test but am running into a number of problems.

Following directions (substituting for react-native-macos): http://facebook.github.io/jest/docs/tutorial-react-native.html

**Comment by ptmt** [comment]
Would you mind to share example repository? Or I can add jest testing example to one of my test projects and share with you.

**Comment by theneverstill** [comment]
The latter would be perfect, thank you!

**Comment by theneverstill** [comment]
The rabbit hole I started to go down arose from the issue:

` Test suite failed to run

    Cannot find module 'Image' from 'setup.js'

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:142:17)`

Unless I included react-native as a dev dependency, I would get this issue.

**Comment by theneverstill** [comment]
I've made a lot of headway here as far as getting Jest running correctly. I had to make my own preprocessor to convert typescript AS WELL AS run babel on non-typescript and also modify jest-react-native setup.js to work with react-native-macos.

Unfortunately, the above still requires the npm install of react-native.

One or more pull requests forthcoming to fix some random issues that have cropped up when making tests.
